pkg/controller/nmstate/nmstate_controller_test.go

```diff
@@ -26,7 +26,7 @@ var _ = Describe("NMState controller reconcile", func() {
 		cl                  client.Client
 		reconciler          ReconcileNMState
 		existingNMStateName = "nmstate"
-		dsNodeSelector      = map[string]string{"selector_1": "value_1", "selector_2": "value_2"}
+		handlerNodeSelector = map[string]string{"selector_1": "value_1", "selector_2": "value_2"}
```

Comment:
so many unrelated changes making this harder to review and spot real issues, tsk tsk tsk.